lib/srv/exec_test.go

```diff
@@ -106,6 +110,8 @@ func TestLoginDefsParser(t *testing.T) {
 }
 
 func newExecServerContext(t *testing.T, srv Server) *ServerContext {
+	t.Parallel()
```

Comment:
I wouldn't put this here because it is a helper function that may be called by a test that shouldn't be run in parallel.

Reply:
Fixed, strange could have swore I ran tests and everything pass before I created the PR.